frontend/pages/index.vue

```diff
@@ -51,7 +51,7 @@
                         v-model="collectionId"
                         :rules="activeJobIsUsingArchive ? collectionRules : gitCollectionRules"
                         label="Collection ID"
-                        class="collection-id-error-text collection-id-incorrect-error-text collection-id-field"
+                        class="collection-id-error-text git-pdf-error-text collection-id-incorrect-error-text git-pdf-incorrect-error-text collection-id-field"
```

Comment:
additional classes are not needed, we should use the same class for locating the same element, ideally just `collection-id-field` here